### DISCUSSION TITLE
Attempt to submit a notice resulted in data corruption

### DISCUSSION THREAD

**Discussion by eduardsmirnov** [OP-TED/eForms-SDK#489] [Q&A] [answered]
Hello, just tried to publish a notice of subtype 1 in a PREVIEW environment using API https://enotices2.preview.ted.europa.eu/esenders/webjars/swagger-ui/index.html#/Notice/submitNotice

Notice id is "744ca70f-e753-4563-a031-a0d742044252-01"
Published as user "publication-test@mercell.com"

Notice passed validation and I see it in Submitted status in eNotices2.

When I export a notice as XML, I see it has corrupted data, one of the Organizations has been removed and a Buyer party got a reference to the wrong organization. Could you check and tell me what happened?

Input XML file is 
[temp2.txt](https://github.com/OP-TED/eForms-SDK/files/11564558/temp2.txt)

Exported XML file is 
[PLANNING_pin-buyer_744ca70f-e753-4563-a031-a0d742044252-01.txt](https://github.com/OP-TED/eForms-SDK/files/11564565/PLANNING_pin-buyer_744ca70f-e753-4563-a031-a0d742044252-01.txt)

**Comment by rouschr** [comment, marked as answer]
Hello, as this is about publishing a notice using enotices2 it would be best to contact the enotices2 helpdesk:

https://enotices2.ted.europa.eu/help/

https://enotices2.ted.europa.eu/help/contact

**Reply by eduardsmirnov** [reply]
@rouschr it is a PREVIEW environment, do you think still the enotices2 helpdesk?
@rousso ?
